ra_aid/__main__.py

```
@@ -906,51 +928,126 @@ def main():
-                    config_repo.set("mcp_use_enabled", False)
+                    logger.info("No MCP servers configured or enabled.")
+                    mcp_enabled = False
+                
+                config_repo.set("mcp_use_enabled", mcp_enabled)
 
                 config_repo.set("cowboy_mode", args.cowboy_mode) # Also add here for non-server mode
 
+
                 # Initialize and register MCP-Use client for cleanup if enabled
-                mcp_use_client_instance = None
+                mcp_use_client_instance: Optional[MCPUseClientSync] = None
```

Ruff (0.8.2)

Undefined name `MCPUseClientSync`

(F821)